### DISCUSSION TITLE
brew doctor

### DISCUSSION THREAD

**Discussion by PratikDhanave** [Homebrew/discussions#1968] [Everyday usage]
Hi community thank you for building amazing brew project. 
I want to know how brew doctor is implemented I think it a very awesome feature. I am wokring on  http://github.com/airshipit/airshipctl project I want to add similar feature airshipctl --doctor which will work similar to brew doctor.

Please guide me with source code of airshipctl --doctor  or point of contact will be really helpful.

Thank you @brew team for amazing  brew project

**Comment by SMillerDev** [comment]
We just have a list of checks: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula_auditor.rb
